src/components/tree-view.tsx

```
@@ -46,87 +42,80 @@ type TreeProps = React.HTMLAttributes<HTMLDivElement> & {
-            ids.push(items[i]!.id);
-            if (walkTreeItems(items[i]!, targetId) && !expandAll) {
-              return true;
-            }
-            if (!expandAll) ids.pop();
+    const ids: string[] = [];
+
+    function walkTreeItems(
+      items: TreeDataItem[] | TreeDataItem,
+      targetId: string,
+    ) {
+      if (items instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)